package.json

```diff
@@ -211,8 +210,5 @@
     "hoist-non-react-statics": "^3.3.0",
     "lodash": "^4.17.4",
     "tslib": "^1.9.3"
-  },
```

Comment:
sewing-kit now uses prettier v2 out the box so we don't need to force the resolution anymore